packages/react-app/src/views/SubmissionReviewView.jsx

```diff
@@ -86,7 +86,12 @@ export default function SubmissionReviewView({ userProvider }) {
     // eslint-disable-next-line
   }, [address]);
 
-  const handleSendChallengeReview = reviewType => async (userAddress, challengeId, comment) => {
+  const handleSendChallengeReview = reviewType => async (badgeId, userAddress, challengeId, comment) => {
+    try {
+      await tx(writeContracts.BuidlBadges.mint(userAddress, badgeId));
+    } catch (e) {
+      console.log("mint tx error:", e);
```

Comment:
If this fails we enter in the catch, log the error, and continue. This means the badge never gets minted and we also don't have a way to retry the mint. I'll add a toast to make this more explicit to the person accepting the challenge but I was thinking we should consider either:
 - Add a way to retry the minting.
 - Not allow to accept a submission if the badge cannot be minted.

The workaround here is to go to the contract with a blockchain explorer and interact with the contract manually.

Reply:
@carletex thoughts?

Reply:
> Not allow to accept a submission if the badge cannot be minted.

I think this should be the way. I did a quick fix on: 9bd0a84b39b00a2c6f9a514977800c41ea14af22